### ISSUE TITLE
Add emojis features

### ISSUE THREAD

**Issue by ankitjain28may** [ankitjain28may/openchat#16]

**Comment by gofaniyi** [comment]
Hi, @ankitjain28may I would like to work on this issue. I have implemented this feature before now on other projects. I will be adding selecting emoji icons (as shown below) and the support to type keyboard shortcuts for the emojis.
![image](https://user-images.githubusercontent.com/11710941/35787106-0586d7ea-0a2c-11e8-9ace-5bf9355f8f98.png)
Let me know if this is ok.

**Comment by ankitjain28may** [comment]
@gofaniyi Great, go ahead..!

**Comment by gofaniyi** [comment]
@ankitjain28may Hi, I am having issues setting up the project on my local machine. I am getting the error below. I will be glad if you can provide a guide/fix. Thanks
`sudo supervisorctl reread
error: <class 'socket.error'>, [Errno 61] Connection refused: file: /System/Library/Frameworks/Python.framework/Versions/2.7/l
ib/python2.7/socket.py line: 575`

**Comment by ankitjain28may** [comment]
@gofaniyi Did u install supervisor ?
If you find problem in running it using supervisor, go through with the comments in the issue [https://github.com/ankitjain28may/openchat/issues/24](https://github.com/ankitjain28may/openchat/issues/24)

**Comment by gofaniyi** [comment]
@ankitjain28may Yes I have installed supervisor. But I went through the comments in issue #24. So I have the server running but I am getting the error below:
![image](https://user-images.githubusercontent.com/11710941/35894593-d307d9ca-0bb3-11e8-8174-6db435629ea8.png)
Kindly help with this.

**Comment by ankitjain28may** [comment]
Are you still getting the same error ?

**Comment by gofaniyi** [comment]
@ankitjain28may I am getting another error. Invalid Authentication. I can successfully launch the app now but then I click on the send button, it gives me the error message I sent in my last comment.

**Comment by ankitjain28may** [comment]
@gofaniyi Try to login again after logout.
